src/web/pager-duty/PagerDutyNotificationDetails.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+import {ReadOnlyFormGroup} from 'components/common';
+
+const PagerDutyNotificationDetails = ({ notification }) => {
+    return (
+        <>
+            <ReadOnlyFormGroup label="Routing Key" value={notification?.config?.routing_key} />
```

Comment:
since you have `notification.config` set to isRequired you can just remove the first conditional

```suggestion
            <ReadOnlyFormGroup label="Routing Key" value={notification.config?.routing_key} />
```

Reply:
Will do